### PR TITLE
[sc-3237] Give direct access to Desktop app download

### PR DESCRIPTION
**Pull request by operramon**

### CODE REVIEW

File: apps/dashboard/src/app/knowledge-box/utils.ts [hunk 1]
```diff
@@ -7,3 +9,36 @@ export const KB_ROLE_TITLES: { [role in KBRoles]: string } = {
...
+  return null;
+}
+
+export function getDesktopAppUrl(platform: 'mac' | 'win' | 'linux'): Observable<string | null> {
+  const filenames = {
+    mac: /.*-universal-mac\.zip$/,
+    win: /.*\.exe$/,
+    linux: /.*\.snap$/,
+  };
+  return fromFetch('https://api.github.com/repos/nuclia/frontend/releases/latest').pipe(
```
Comment (mpellerin42) [inline apps/dashboard/src/app/knowledge-box/utils.ts:31]: you're using this URL here and on `knowledge-box-home.component.ts` as well, you could export it in a const in utils.ts, and use this const everywhere instead.

File: apps/dashboard/src/app/knowledge-box/utils.ts [hunk 2]
```diff
@@ -7,3 +9,36 @@ export const KB_ROLE_TITLES: { [role in KBRoles]: string } = {
...
+  }
+  return null;
+}
+
+export function getDesktopAppUrl(platform: 'mac' | 'win' | 'linux'): Observable<string | null> {
+  const filenames = {
+    mac: /.*-universal-mac\.zip$/,
+    win: /.*\.exe$/,
+    linux: /.*\.snap$/,
+  };
```
Comment (mpellerin42) [inline apps/dashboard/src/app/knowledge-box/utils.ts:30]: usually it's better to define regexp once for all (it's better for performances), so you can move this const outside of this function (no need to export it though).

**Review by mpellerin42 — APPROVED** [review]

File: apps/dashboard/src/app/knowledge-box/utils.ts (hunk 1 above)
Comment (operramon) [reply apps/dashboard/src/app/knowledge-box/utils.ts]: fixed

File: apps/dashboard/src/app/knowledge-box/utils.ts (hunk 2 above)
Comment (operramon) [reply apps/dashboard/src/app/knowledge-box/utils.ts]: fixed 👍

**Review by mpellerin42 — APPROVED** [review]

**Pull request merged**